lib/plugin.rb

```diff
@@ -21,18 +23,32 @@ def update
   private
 
   def zapp_plugin
-    get_request(plugins_url, request_params)
+    plugin_candidates = get_request(plugins_url, request_params)
       .body
-      .find { |p| p["name"] == @name || p["external_identifier"] == @identifier }
+      .select do |p|
+        p["name"] == @name || identifier_matches?(p)
+      end
+
+    return nil if plugin_candidates.count == 0
```

Comment:
you can just return instead of return `nil`

Reply:
Ok